src/commands/utils/modules.ts

```diff
@@ -45,6 +51,23 @@ class JahiaUtilsModule extends Command {
     const dependencies: string[] = flags.dependencies.split(',')
 
     const jahiaFullUrl = flags.jahiaUrl.slice(-1) === '/' ? flags.jahiaUrl : flags.jahiaUrl + '/'
+    for (let i = 0; i < flags.timeout; i++) {
+      let out = false
+      setTimeout(() => {
```

Comment:
yes setTimeout gives you the same result, but I find that adding a sleep of 1s after the call improves readability.

https://github.com/Jahia/jahia-reporter/blob/main/src/utils/sleep.ts